src/pages/CategoriesPage.js

```diff
@@ -0,0 +1,15 @@
+import { useSelector } from 'react-redux';
+
+const Categories = () => {
+  const { categories, message } = useSelector((state) => state.categories);
+
+  return (
+    <div>
+      <div />
+      <div><p>{categories}</p></div>
+      <div><p>{message}</p></div>
+    </div>
+  );
+};
```

Comment:
- [ ] On the Category page, we should display only a button with the text "Check status". Kindly fix it. Also, prove the nav there so we can easily get back to the home page. Right now it's showing a blank page.

Reply:
Done